crates/solidity/testing/snapshots/cst_output/NumericExpression/hex_uppercase_prefix/generated/0.5.0.yml

```diff
@@ -5,12 +5,15 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected end of input.
+    Error: Expected DaysKeyword or EtherKeyword or FinneyKeyword or HoursKeyword or MinutesKeyword or SecondsKeyword or SzaboKeyword or WeeksKeyword or WeiKeyword.
```

Comment:
this exposed a bug in our grammar where `0` followed by `x` should not be matchd as a `DecimalLiteral`, but as a `HexLiteral` .. I will fix it in my current PR.

But I wonder what changed the output in this PR though? Are we scanning input differently somehow?

Reply:
This is an uppercase `X`, which from 0.5.0 onwards is not a valid hex literal